index.html

```diff
@@ -273,5 +274,49 @@ <h3>Skills</h3>
         <hr />
       </ul>
     </div>
+    <footer class="contact">
+      <section class="background2">
+        <form
+          class="form"
+          action="https://formspree.io/f/xdovrwzl"
+          method="POST"
+        >
+          <div class="section3">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it!
+            </p>
+            <input
+              id="name"
+              class="name"
+              name="name"
+              type="text"
+              placeholder="Lorem Ipsum"
+              maxlength="30"
+            />
+            <input
+              id="email"
+              class="email"
+              name="email"
+              type="email"
+              placeholder="loremipsum@gmail.com"
+            />
+            <textarea
+              id="text"
+              class="text"
+              type="text"
+              name="message"
+              placeholder="Write your message here..."
+              maxlength="500"
+            ></textarea>
```

Comment:
Great job on the input fields! 👏 Here are some suggestions to further enhance them:

1. Required Attribute: It's important to include the "required" attribute in all the input fields. This ensures that users cannot submit the form without filling in these required fields. Please update the input fields accordingly. ⚠️🔒

2. Placeholder Styles: While the placeholders are present, they could be improved to align with the design. Please update the styles of the placeholders to match the design specifications. 🎨🖊️

4. Meaningful Placeholders: The placeholders currently used in the input fields could be more meaningful. Consider updating them to provide clear and specific instructions or examples of the expected input. This will assist users in understanding the purpose of each field. ✍️💡
![image](https://github.com/peter-mwovi/portfolio_setup/assets/55840999/4a5f2d94-3155-489c-8109-78c94db7ba41)

5. Input Field Padding: Ensure that the padding of the input fields is consistent with the design. Please adjust the padding to align with the specified design guidelines. 📐🔲
![Frame 40](https://github.com/peter-mwovi/portfolio_setup/assets/55840999/3e548c9d-a372-48b2-b60f-fc5b00e16e2c)

By implementing these suggestions, you'll enhance the usability and visual appeal of the input fields, creating a better user experience. Keep up the great work! If you have any questions, feel free to ask. 🚀😊